### PR TITLE
Images for models

### PR DESCRIPTION
**Pull request by cdoyle-temple**
Please merge this so I can work on variants in master.

### CODE REVIEW

File: app/fields/building_photo_field.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require "administrate/field/base"
+
+class BuildingPhotoField < Administrate::Field::Base
+  def to_s
+	    data
+  end
```
Comment (nomadicoder) [inline app/fields/building_photo_field.rb]: What will this look like if we do `building.photo.to_s`? Should it be something else, like the name of the file?

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -104,4 +114,4 @@ form {
 }
 .no-padding {
 	padding: 0;
-}
+}
```
Comment (nomadicoder) [inline app/assets/stylesheets/application.scss]: Please fix - add a new line.

File: app/fields/person_photo_field.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+require "administrate/field/base"
+
+class PersonPhotoField < Administrate::Field::Base
+  def to_s
+	    data
+  end
```
Comment (nomadicoder) [inline app/fields/person_photo_field.rb]: Same comment as the previous `to_s`.

File: app/fields/space_photo_field.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+require "administrate/field/base"
+
+class SpacePhotoField < Administrate::Field::Base
+  def to_s
+	    data
+  end
```
Comment (nomadicoder) [inline app/fields/space_photo_field.rb]: And here too.

File: app/fields/building_photo_field.rb (hunk 1 above)
Comment (cdoyle-temple) [reply app/fields/building_photo_field.rb]: building_photo_field has now been deleted. we're not using photos for buildings per cynthia.

File: app/fields/person_photo_field.rb (hunk 3 above)
Comment (cdoyle-temple) [reply app/fields/person_photo_field.rb]: Has no visible effect, so am changing as requested.

**Pull request merged**